### PR TITLE
Fix running Servo in CI

### PR DESCRIPTION
**Pull request by jgraham**
Example run https://community-tc.services.mozilla.com/tasks/groups/BtQNCaJjSv2zao9zf0T1Gg

wdspec tests still fail; these seem to be broken for unrelated reasons.

### CODE REVIEW

File: tools/ci/taskcluster-run.py [hunk 1]
```diff
@@ -10,8 +10,7 @@
 
 browser_specific_args = {
     "firefox": ["--install-browser"],
-    "servo": ["--install-browser"]
-
+    "servo": ["--install-browser", "--processes=12"]
```
Comment (gsnedders) [inline tools/ci/taskcluster-run.py]: Note we know this causes test flakiness, so we might well not want this for the same reason as why we don't do it for other browsers.

Comment (gsnedders) [reply tools/ci/taskcluster-run.py]: Also just a bit weary about running one browser in a different configuration than the others.

**Comment by jgraham** [comment]
https://community-tc.services.mozilla.com/tasks/groups/HSHMOGm4SdaJLa6-U2KI7A is a complete run.

File: tools/ci/taskcluster-run.py (hunk 1 above)
Comment (jgraham) [reply tools/ci/taskcluster-run.py]: Without this I think it's going to be too slow to run. I intentionally copied the servo CI configuration, which I assume they are happy with.

I agree we could probably do better here, but I think landing this as-is gives us the chance to figure out what better looks like.

Comment (gsnedders) [reply tools/ci/taskcluster-run.py]: And that's because the product is Servo and not ServoDriver, so it's restarting the browser all the time.

**Review by gsnedders — APPROVED** [review]

**Pull request merged**